### PR TITLE
Update included code block languages + misc formatting

### PR DESCRIPTION
**Pull request by siegerts**
*Issue #, if available:*

*Description of changes:*

- Add all missing language references for fenced code languages + remove unused 
- Adjust lang refs where needed
- minor formatting

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/cli/usage/containers.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ This `docker-compose.yml` file would be placed in your `./amplify/backend/api/<n
    Dockerfile
    package.json
    index.js
-/python
+ /python
```
Comment (attilah) [inline docs/cli/usage/containers.md:150]: is the whitespace intentional here?

**Review by attilah — APPROVED** [review]
LGTM, 1 comment about a whitespace change if we need that or not

File: docs/cli/usage/containers.md (hunk 1 above)
Comment (siegerts) [reply docs/cli/usage/containers.md:150]: Yes, the live version is _missing_ to indicate the `/python` dir at the same level as the `/express` dir.

ref: https://docs.amplify.aws/cli/usage/containers#deploy-multiple-containers

File: docs/cli/usage/upgrade.md [hunk 2]
```diff
@@ -11,11 +11,12 @@ Follow the steps below to update to the latest version of the CLI:
     ```bash
     npm install -g @aws-amplify/cli
     ```
-2. Verify the successful installation of the latest CLI version by entering the following command in the CLI
+2. Verify the successful installation of the latest CLI version by entering the following command in the CLI:
     ```bash
     amplify -v
     ```
-    and confirm the installed version of the Amplify CLI. You can find the latest version of the CLI here - https://www.npmjs.com/package/@aws-amplify/cli
+    Confirm the installed version of the Amplify CLI. You can find the latest version of the CLI here - https://www.npmjs.com/package/@aws-amplify/cli.
```
Comment (jamesonwilliams) [inline docs/cli/usage/upgrade.md:18]:
```suggestion
    Confirm the installed version of the Amplify CLI. You can find the latest version of the CLI on [npmjs.com](https://www.npmjs.com/package/@aws-amplify/cli).
```

**Review by renebrandel — APPROVED** [review]

File: docs/guides/api-rest/python-api.md [hunk 3]
```diff
@@ -48,12 +48,12 @@ def handler(event, context):
   current_time = datetime.datetime.now().time()
 
   body = {
-      "message": "Hello, the current time is " + str(current_time)
+      'message': 'Hello, the current time is ' + str(current_time)
```
Comment (jamesonwilliams) [inline docs/guides/api-rest/python-api.md:51]: Perhaps a format string, instead?

https://matthew-brett.github.io/teaching/string_formatting.html

**Review by jamesonwilliams — APPROVED** [review]

**Review by dabit3 — APPROVED** [review]

File: docs/guides/api-rest/python-api.md (hunk 3 above)
Comment (siegerts) [reply docs/guides/api-rest/python-api.md:51]: I'm not opposed, but, in that case I'd recommend reformatting all of the py docs (~10) with with either `black`/`autopep` (sep PR) to make it standard across examples along with any updated language constructs.

**Pull request merged**